### ISSUE TITLE
Caching values based on component inner content static sections

### ISSUE THREAD

**Issue by tokrug** [wttech/markuply#2]
**Scenario**

Default inner content section of a component is treated as a Pebble template. Thatsection  is fully static which means it does not contain any component placeholders. Because of that the Pebble template should be compiled only once and then reused within the context of the RenderFunction. Coupled with RenderFunction cache it should improve performance.

**To be decided**

If such feature is in fact important. Please describe your use case in comments below if you'd like this feature to be implemented.

**Out of scope**

Same scenario but inner content section contains a component placeholder with component logic dependent only on component props.
